### DISCUSSION TITLE
How to change the name of the main activity and add more activity?

### DISCUSSION THREAD

**Discussion by ccinsz** [beeware/briefcase#1344] [Q&A]
Hi, now the main activity of the app is org.beeware.android.MainActivity, how to change it? And if I want to add more page, how to add other activity?

**Comment by mhsmith** [comment]
> Hi, now the main activity of the app is org.beeware.android.MainActivity, how to change it?

The class name of the activity is currenty hard-coded. However, you can still change:
* The visible activity title, by assigning to [`Window.title`](https://toga.readthedocs.io/en/stable/reference/api/window.html#toga.window.Window.title).
* The application ID, by editing the `bundle` setting in pyproject.toml, then re-running `briefcase create android`.

> And if I want to add more page, how to add other activity?

 Toga does not currently support multiple activities on Android. But if you want your app to switch between multiple screens, you can achieve that by assigning to [`Window.content`](https://toga.readthedocs.io/en/stable/reference/api/window.html#toga.window.Window.content).

**Reply by ccinsz** [reply]
When I change the window.title, it doesn't work. like this  self.main_window = toga.MainWindow(title="kkkk") or def main():
    return demo0529("kkkk")

And it can only change the package name in pyproject.toml.

**Reply by mhsmith** [reply]
It looks like the `title` constructor parameter is currently unimplemented on Android, but you can still assign to the property of the same name.
